src/slack/slack.js

```diff
@@ -789,29 +711,29 @@ export class Slack {
 
       // for wincent, don't go into spinoff selection
       if (this.variant === 'wincent') {
-        await submitOrder(order.isHome ? MESSAGES.home.company : MESSAGES.office.company, true)
-        return true
+        order.messages = order.isHome ? MESSAGES.home.company : MESSAGES.office.company
```

Comment:
again, when return was removed, the code for vacuumlabs and test variant would run for wincent. need `else`